pupil_src/shared_modules/vis_eye_video_overlay.py

```diff
@@ -339,12 +339,16 @@ def on_click(self, pos, button, action):
             if not self.eye1.on_click(
```

Comment:
if you wanted to, you could change the `action == 1` above to `action == glfw.GLFW_PRESS`

Reply:
Adapted that in the new version